requirements.txt

```diff
@@ -1,6 +1,6 @@
 joblib
-numpy
+numpy<1.20.0
```

Comment:
We can't use any new numpy with mabwiser?

Reply:
@filip-michalsky is the functionality no longer available, or does it just raise a deprecation warning?

Reply:
it raises a deprecation warning.

Reply:
We don't need to have a version restriction for a deprecation warning, that just means with some later version of numpy it won't be compatible

Reply:
first of all, disclaimer: I am not close to this work so my comments might be off.. 

I am not comfortable with an extremely strong <1.2 requirement. That just forbids the entire library to benefit from any upcoming numpy version.. 

If possible, and if we really want to avoid depreciation warning, we shall fix the code instead and make the jump to new numpy versions. 

If they are deprecating something, I am sure there is a (better) alternative now. We shall uptake that, instead of restricting ourselves to the past, no?

Reply:
Serdar, per the discussion below, the issue is that if we require sklearn < 0.24, and a user has numpy = 1.20, sklearn has the np.dytpe code that is raising the deprecation warnings